lib/grpc/GrpcService.ts

```diff
@@ -21,6 +21,7 @@ const createOrder = (order: StampedOrder) => {
   grpcOrder.setCreatedAt(order.createdAt);
   grpcOrder.setId(order.id);
   if (isOwnOrder(order)) {
+    grpcOrder.setHold(order.hold ? order.hold : 0);
```

Comment:
`hold` is never undefined in the latest code so you can just set this equal to `order.hold`.